host/examples/parsec_execute/src/main.rs

```diff
@@ -0,0 +1,108 @@
+//! Provision and execute parsec.
+
+use anyhow::anyhow;
+use log::{error, info};
+use utils::attestation;
+use transport::{messages::{Request, Response, Status}, session::Session};
+
+use std::fs;
+use std::fs::File;
+use std::io::Read;
+
+const DPU_SERVER_URL: &str = "127.0.0.1:6666";
+const ATTESTATION_SERVER_URL: &str = "127.0.0.1:3010";
+
+const PARSEC_APP_NAME: &str = "parsec_app";
```

Comment:
This its the parsec client's binary, right? Your model makes it difficult to understand and reproduce what code is actually provisioned and executed on the remote target.
How about you push the parsec-tool's binary instead and execute it over the CLI? Should solve most of your problems